src/integration_free-fleet.md

```diff
@@ -1,4 +1,8 @@
-# Free Fleet
+# Legacy - Free Fleet
+
+<div class='warning'>
```

Comment:
This `warning` class div doesn't have any effect in GitHub's markdown preview, but I've confirmed that it does render correctly in the fully generated mdbook 👍

Reply:
oh yeah, I realized the usual github markdown warning didn't work, but thankfully `mdbook` has some [docs](https://rust-lang.github.io/mdBook/format/mdbook.html#classwarning) about it